prow/plugins/lgtm/lgtm.go

```diff
@@ -341,6 +341,17 @@ func handlePullRequest(log *logrus.Entry, gc githubClient, config *plugins.Confi
 
 	// rc.repo.Owner.Login, rc.repo.Name)
 	opts := optionsForRepo(config, org, repo)
+	if opts.StickyForCollaborators {
+		if ok, merr := gc.IsCollaborator(org, repo, pe.PullRequest.User.Login); merr == nil && ok {
```

Comment:
This is probably better as `trigger.trustedUser()`:

https://github.com/kubernetes/test-infra/blob/2264543fd4ad973cfb838a9b4a17cf37dea81f76/prow/plugins/trigger/trigger.go#L142